### DISCUSSION TITLE
Cases - assignee, status - not found in kibana

### DISCUSSION THREAD

**Discussion by jakko10** [Security-Onion-Solutions/securityonion#9103] [General] [answered]
Hi,

If I create a case and set assignee + status, is it possible to somehow search these cases from elastc dev tools -> console?

So far I can find all the events with no problem but not the so_case.assigneeId, so_case.status values.
Is it even possible?

BR
Jakko

**Comment by jakko10** [comment]
Or is it possible to query from so-case? I'm guessing the data is there but I just don't know how to approach it.
The aim is to query cases and verify the values of so_case.assigned and so_case.status, etc.

**Comment by dougburks** [comment, marked as answer]
The main Cases page shows the `so_case.status` and `so_case.assigneeId` columns:
![image](https://user-images.githubusercontent.com/1659467/201083560-b61b76d8-304d-4779-9b74-f02effdd604f.png)

You should be able to click on a value in those columns to get a context menu that allows you to include, exclude, etc. as described on the Dashboards page:
https://docs.securityonion.net/en/2.3/dashboards.html#context-menu

If you need even more features in the main Cases interface, you can enable advanced interface features:
https://docs.securityonion.net/en/2.3/cases.html#options

**Reply by jakko10** [reply]
Thank you for the answer!

The main goal is to query these values via "sudo so-elasticsearch-query" though.

**Reply by jakko10** [reply]
Or well, the main goal is to query if the case has been created, then what's the case title, who's been assigned to the case and the status of the case.

**Reply by dougburks** [reply]
> The main goal is to query these values via "sudo so-elasticsearch-query" though.

Cases stores all of its data in Elasticsearch and so you should be able to query using `sudo so-elasticsearch-query`. For more information, please see https://docs.securityonion.net/en/2.3/cases.html#data.

**Reply by jakko10** [reply]
Do you mind giving me some examples I'm kind of stuck with this.
This doesn't give me anything back: `sudo so-elasticsearch-query _index:"*:so-case" | jq`
This does: `sudo so-elasticsearch-query so-case/_search | jq`
But the so_case.status and so_case.assigned are missing. 
What am I doing wrong?

**Reply by jakko10** [reply]
Actually I think I was blind, the status is there and I guess the query does not display the actual assigned username, the assignedId is there, so I guess I'll work with that.

**Reply by jakko10** [reply]
One last question regarding the cases. It's possible to add comments to the case, will the case comment appear in the elastic too or is it stored somewhere else?

**Reply by dougburks** [reply]
As I mentioned in my last reply, Cases stores all of its data in Elasticsearch.

**Reply by jakko10** [reply]
Yes, but where? I've literally spent a day trying to find the data from elastic, the documentation does not cover this.
Please could you provide some sort of an example?

**Reply by dougburks** [reply]
Here's an example for you. On the left, you can see that I created a case and added 2 comments. On the right, you can see the output of `sudo so-elasticsearch-query so-case/_search | jq` showing the case and the 2 comments.

![image](https://user-images.githubusercontent.com/1659467/201205386-1485556f-8e3d-4b22-b199-8ef228738acf.png)

**Reply by jakko10** [reply]
Hehe, I just found more-or-less the same solution, the example helped me to better understand it though.
Thank you!
